ui/raidboss/data/07-dt/trial/necron-ex.txt

```diff
@@ -0,0 +1,260 @@
+125.3 "Aetherblight" Ability { id: ["AE40", "B07F", "B080", "B081"], source: "Necron" }
+125.4 "Shockwave" Ability { id: ["AE0F", "AE10"], source: "Necron" }
+
+# Grand Cross 1
+140.2 "Grand Cross (damage)" Ability { id: "AE18", source: "Necron" }
+148.3 "Grand Cross (puddle)" #Ability { id: "AE1B", source: "Necron" }
+150.3 "Grand Cross (puddle)" #Ability { id: "AE1B", source: "Necron" }
+150.7 "Grand Cross (line)" #Ability { id: "AE19", source: "Azure Aether" }
+152.7 "Grand Cross (line)" #Ability { id: "AE19", source: "Azure Aether" }
+154.3 "Shock (tower)" Ability { id: "AE1D", source: "Necron" }
+154.3 "Grand Cross (spread)" Ability { id: "AE1C", source: "Necron" }
+157.3 "Grand Cross (line)" Ability { id: "AE19", source: "Azure Aether" }
+161.3 "Grand Cross (line)" Ability { id: "AE19", source: "Azure Aether" }
+164.0 "Grand Cross (line)" Ability { id: "AE19", source: "Azure Aether" }
+165.2 "Shock (tower)" Ability { id: "AE1D", source: "Necron" }
+165.2 "Grand Cross (spread)" Ability { id: "AE1C", source: "Necron" }
+172.2 "Grand Cross (puddle)" #Ability { id: "AE1B", source: "Necron" }
+173.9 "Grand Cross (intercards)" Ability { id: "AE1A", source: "Azure Aether" }
+174.2 "Grand Cross (puddle)" #Ability { id: "AE1B", source: "Necron" }
+184.1 "Neutron Ring (cast)" Ability { id: "AE1F", source: "Necron" }
+186.8 "Neutron Ring (damage)" Ability { id: "AE20", source: "Necron" }
+
+# Adds Phase
+199.5 "Fear of Death (puddles)" Ability { id: "AE21", source: "Necron" }
+206.6 "Choking Grasp (lines)" duration 37.5 #Ability { id: "AE23", source: "Icy Hands" }
+207.4 "Fear of Death (puddles)" Ability { id: "AE21", source: "Necron" }
+210.7 "Muted Struggle?" #Ability { id: "AE22", source: "Beckoning Hands" }
+215.5 "Fear of Death (puddles)" Ability { id: "AE21", source: "Necron" }
+221.2 "Muted Struggle?" #Ability { id: "AE22", source: "Beckoning Hands" }
+
+# Solo Phase
+250.8 "Darkness of Eternity" Ability { id: "AE24", source: "Necron" }
+257.2 "Darkness of Eternity" Ability { id: "AE25", source: "Necron" }
+260.9 "Inevitability" Ability { id: "AE27", source: "Necron" }
+279.0 "Choking Grasp" Ability { id: "AE28", source: "Icy Hands" }
+284.1 "Choking Grasp" Ability { id: "AE28", source: "Icy Hands" }
+289.3 "Choking Grasp" Ability { id: "AE28", source: "Icy Hands" }
+
+# P2 Start
+321.0 "Specter of Death" Ability { id: "AE3E", source: "Necron" } window 321,30
+329.1 "The End's Embrace" Ability { id: "AE36", source: "Necron" }
+335.2 "Choking Grasp" #Ability { id: "AE17", source: "Icy Hands" }
+337.1 "Cold Grip" Ability { id: "AE44", source: "Necron" }
+338.8 "Existential Dread" Ability { id: "AE0B", source: "Necron" }
+
+# Relentless Reaping 1
+360.5 "Relentless Reaping" Ability { id: "AE14", source: "Necron" }
+366.6 "Crop Rotation" Ability { id: "AE42", source: "Necron" }
+377.7 "The Second Season/The Fourth Season" Ability { id: ["B06F", "B070"], source: "Necron" }
```

Comment:
Same for this as for Blight, a `replaceText` of `Second/Fourth Season` might make sense.